modules/bullet/space_bullet.cpp

```diff
@@ -542,7 +542,6 @@ void SpaceBullet::remove_all_collision_objects() {
 }
 
 void onBulletPreTickCallback(btDynamicsWorld *p_dynamicsWorld, btScalar timeStep) {
-	static_cast<SpaceBullet *>(p_dynamicsWorld->getWorldUserInfo())->flush_queries();
```

Comment:
Should we de-register the callback if we don't use it anymore? @AndreaCatania

Reply:
Updated to also remove the pre-tick callback.